File: src/backend/studio-reservation/src/main/java/org/taonaw/studio_reservation/usecase/command/equipment/registerEquipment/RegisterEquipmentCommand.java
package org.taonaw.studio_reservation.usecase.command.equipment.registerEquipment;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import org.taonaw.studio_reservation.domain.model.equipment.EquipmentName;
import org.taonaw.studio_reservation.domain.model.equipment.EquipmentStockCount;
import org.taonaw.studio_reservation.domain.model.equipmentCategory.EquipmentCategoryId;

@Getter
@Builder
@AllArgsConstructor
public class RegisterEquipmentCommand {
    private EquipmentName name;
    private EquipmentCategoryId categoryId;
    private EquipmentStockCount stockCount;
}

File: src/backend/facility/src/main/kotlin/org/taonaw/facility/usecase/equipment/registerEquipment/RegisterEquipmentCommand.kt
package org.taonaw.facility.usecase.equipment.registerEquipment

import org.taonaw.facility.domain.model.equipment.EquipmentName
import org.taonaw.facility.domain.model.equipment.EquipmentRentalFee
import org.taonaw.facility.domain.model.equipment.EquipmentStocks
import org.taonaw.facility.domain.model.equipmentCategory.EquipmentCategoryId

data class RegisterEquipmentCommand(
        val equipmentName: EquipmentName,
        val equipmentCategoryId: EquipmentCategoryId,
        val equipmentRentalFee: EquipmentRentalFee,
        val equipmentStocks: EquipmentStocks) {
}